File: app/src/main/java/com/example/ptakim/MainActivity.java
package com.example.ptakim;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    private  Integer NumOfWords = 20;
    private TextView textView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        textView  = findViewById(R.id.NumOfWorlds);

    }

    public   void  minus(View view){
        if(NumOfWords>20){
            NumOfWords = NumOfWords - 5;
            textView.setText(NumOfWords.toString());

        }
    }
    public   void  plas(View view){
        if(NumOfWords<=35){
            NumOfWords = NumOfWords +5;
            textView.setText(NumOfWords.toString());
        }
    }
    public   void  Start(View view){
        Intent intent = new Intent(this,CategoriesMenu.class);
        intent.putExtra("NumOfWorlds",NumOfWords);

        startActivity(intent);
    }
}

File: app/src/main/java/com/example/ptakim/MainActivity.kt
package com.example.ptakim

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    private var NumOfWords = 20
    private var textView: TextView? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        textView = findViewById(R.id.NumOfWorlds)
    }

    operator fun minus(view: View?) {
        if (NumOfWords > 20) {
            NumOfWords = NumOfWords - 5
            textView!!.text = NumOfWords.toString()
        }
    }

    fun plas(view: View?) {
        if (NumOfWords <= 35) {
            NumOfWords = NumOfWords + 5
            textView!!.text = NumOfWords.toString()
        }
    }

    fun Start(view: View?) {
        val intent = Intent(this, CategoriesMenu::class.java)
        intent.putExtra("NumOfWorlds", NumOfWords)
        startActivity(intent)
    }
}